Travis-CI fails sometimes

```
  <SplitButton>
    ✔ should open the menu when dropdown button is clicked
    ✔ should not open the menu when other button is clicked
    ✔ should invoke onClick when SplitButton.Button is clicked (prop)
    ✖ should not invoke onClick when SplitButton.Toggle is clicked (prop)
    ✔ should not invoke onClick when SplitButton.Toggle is clicked (prop) 
  ✖ "after each" hook for "Should pass disabled to both buttons"
Finished in 6.264 secs / NaN secs @ 19:12:32 GMT+0000 (Coordinated Universal Time)
SUMMARY:
✔ 351 tests completed
✖ 2 tests failed
FAILED TESTS:
  <SplitButton>
    ✖ should not invoke onClick when SplitButton.Toggle is clicked (prop)
      Firefox 56.0.0 (Linux 0.0.0)
    uncaught exception: AssertionError: expected 'next' to equal 'prev' (:0)
  ✖ "after each" hook for "Should pass disabled to both buttons"
    Firefox 56.0.0 (Linux 0.0.0)
  console.error.expected is undefined
  ./test/index.js/<@webpack:///test/index.js:63 <- test/index.js:158200:7
```

Example: https://travis-ci.org/react-bootstrap/react-bootstrap/builds/518914585?utm_source=github_status&utm_medium=notification

--- a/.travis.yml
+++ b/.travis.yml
@@ -16,7 +16,7 @@ cache:
 
 addons:
   chrome: stable
-  firefox: stable
+  firefox: latest
 
 before_install:
   - export DISPLAY=:99.0

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,7 @@
 sudo: false
 
+dist: xenial
+
 language: node_js
 node_js:
   - "10"

--- a/.travis.yml
+++ b/.travis.yml
@@ -16,6 +16,10 @@ cache:
     - node_modules
     - www/node_modules
 
+before_install:	
+  - export CHROME_BIN=chromium-browser	
+  - export DISPLAY=:99.0
+
 install:
   - yarn bootstrap
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,10 +1,8 @@
 sudo: false
 
-dist: xenial
-
 language: node_js
 node_js:
-  - "10"
+  - lts/*
 
 env:
   - BROWSER=ChromeCi
@@ -16,9 +14,13 @@ cache:
     - node_modules
     - www/node_modules
 
-before_install:	
-  - export CHROME_BIN=chromium-browser	
+addons:
+  chrome: stable
+  firefox: stable
+
+before_install:
   - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
 
 install:
   - yarn bootstrap

--- a/.travis.yml
+++ b/.travis.yml
@@ -16,11 +16,6 @@ cache:
     - node_modules
     - www/node_modules
 
-before_install:
-  - export CHROME_BIN=chromium-browser
-  - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
-
 install:
   - yarn bootstrap
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,3 @@
-sudo: false
-
 language: node_js
 node_js:
   - lts/*

